Modules.md

```diff
@@ -32,6 +32,13 @@ in a Web environment (more details [below](Modules.md#integration-with-es6-modul
 WebAssembly modules are defined independently of JavaScript and do not require
 the host environment to include a JavaScript VM.
 
+From a C/C++ perspective, a WebAssembly module is not a single ```.o``` file
+or a single class implementation but rather the
+equivalent of a C/C++ program with all its ```.o``` files linked together. At
+runtime, a module can be loaded by a runtime system, which would load at runtime
+the module. If the module has a start definition, that definition gets called
+before any other function from the module.
```

Comment:
It's not clear that we actually guarantee this, in the case of cyclic module dependencies. But that said, I wonder if saying "executable" above is enough clarification, and we don't need to mention the module start function separately here. It's covered in Modules.md, and is a sufficiently low-level detail that this high-level C/C++-oriented doc may not need.